modules/common/services/xdghandlers.nix

```diff
@@ -35,16 +35,16 @@ let
   xdgOpenFile = pkgs.writeShellScriptBin "xdgopenfile" ''
```

Comment:
Change this to writeShellApplication, then the packages are passed as inputs and you don't need to use app path, just binary name e.g. '`realpath`' vs `pkgs.coreutils/bin/...`

Reply:
resolved